server/sonar-server/src/test/java/org/sonar/server/rule/ws/UpdateActionTest.java

```diff
@@ -310,6 +310,27 @@ public void fail_to_update_custom_when_description_is_empty() {
       .execute();
   }
 
+  @Test
+  public void throw_IllegalArgumentException_if_trying_to_update_builtin_rule_description() throws Exception {
+    logInAsQProfileAdministrator();
+    RuleDefinitionDto templateRule = db.rules().insert(
+      r -> r.setRuleKey(RuleKey.of("java", "S001")),
+      r -> r.setIsTemplate(true),
```

Comment:
A template rule is not necessarily a built-in rule.
Please simply the creation of the rule by only using `db.rules().insert()`, not need to override anything

Reply:
done